Prose/ProseLib/Sources/ProseCore/ProseClient.swift

```diff
@@ -0,0 +1,108 @@
+import Foundation
```

Comment:
Could you please add a `Logger` for this new target? Like https://github.com/prose-im/prose-app-macos/pull/67/files#diff-c796bd38de6789a1cfe0baf42f8db2614aa16ce4ecbd338c0363599865e7a5c5. I know we don't use it yet, so you'll probably tell me we'll add it when we need it, so do as you want 😉

Reply:
I promise to do so.